.github/workflows/coverage.yml

```diff
@@ -1,6 +1,7 @@
 name: CI
 
 on:
+  workflow_dispatch:
```

Comment:
Sneaky undocumented change

Reply:
Yeah that was to enable testing but also handy to have for tries on non-PR branches etc